reason_4.0/lib/core/minisite_templates/modules/social_account/views/profile_links.php

```diff
@@ -37,7 +37,7 @@ function get()
 			foreach ($profile_links as $id => $link)
 			{
 				$space = (isset($space)) ? " " : "";
-				$str .= $space . '<a href="'.$link['href'].'"><img src="'.$link['icon'].'" alt="'.$link['text'].'" /></a>';	
+				$str .= $space . '<a href="'.$link['href'].'"><img src="'.$link['icon'].'" alt="'.$link['text'].'" title="'.$link['name'].'" /></a>';	
```

Comment:
I believe that it is more conventional to put titles on anchors rather than images.

All of these should also be using htmlspecialchars() around the values -- if any of these values have quotes it would break the output.